packages/commonwealth/server/routes/createThread.ts

```diff
@@ -444,6 +445,7 @@ const createThread = async (
     isCustomDomain: null,
   });
 
+  await recomputeValidateChainCache(models, chain.id);
```

Comment:
Why are we recomputing here?

Reply:
sorry, let me rework and verify this one!
Will update here, thanks!